modules/modules.nix

```diff
@@ -30,6 +30,7 @@ let
     ./misc/fontconfig.nix
     ./misc/gtk.nix
     ./misc/lib.nix
+    ./misc/mozilla-messaging-hosts.nix
```

Comment:
I think @kira-bruneau was working on breaking up the firefox module into separate files or something. Wondering if we have a better location for this.